docs/migrating_versions/index.rst

```diff
@@ -0,0 +1,12 @@
+.. _migrating_versions:
+
+Version Migration Guide
+=======================
+
+This section contains pages designed to help navigate major
+changes to the Fairlearn API.
+
+.. toctree::
+   :maxdepth: 1
+
+   migrating_to_v050
```

Comment:
Perhaps one day we'll make it to double digit versions of some kind. Then it would be preferable to have the major minor patch versions separated in some way.

Reply:
Something like `_0_5_0` (although I hope by that point we will be having rather fewer breaking changes, and also have dropped support for older versions)?